wordpress/wp-admin/css/code-editor-rtl.css

```
@@ -0,0 +1,77 @@
+.wp-core-ui .CodeMirror-lint-marker-error,
+.wp-core-ui .CodeMirror-lint-marker-warning {
+	cursor: help;
+}
+
+.wrap .CodeMirror-lint-marker-multiple {
+	position: absolute;
+	top: 0;
+}
+
+.wrap [class*="CodeMirror-lint-marker"]:before {
+	font: normal 18px/1 dashicons;
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)